tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
You can have a better tsconfig I think. This is from one of my repos.
Most importantly, add the `outDir` so that the built JS files are inside the `dist` or `build` (whatever you wanna name it) folder.

```json
{
    "compilerOptions": {
        "lib": [
            "es2019"
        ],
        "module": "commonjs",
        "moduleResolution": "node",
        "esModuleInterop": true,
        "noImplicitAny": true,
        "preserveConstEnums": true,
        "forceConsistentCasingInFileNames": true,
        "resolveJsonModule": true,
        "outDir": "dist",
        "declaration": true
    },
    "include": [
        "src/**/*.ts"
    ],
    "exclude": [
        "src/**/*.spec.ts",
        "node_modules",
        "dist"
    ]
}
```